### DISCUSSION TITLE
Add Project beta's custom issue-fields to API

### DISCUSSION THREAD

**Discussion by jozydapozy** [community/community#14276] [Projects and Issues]
When I add an issue to a Project (Beta), I'm able to assign custom fields to issues. For example a Due date field.
This field should be present in the Github API when requesting an issue

**Comment by michaelbromley** [comment]
Looks like this would be part of this roadmap item: https://github.com/github/roadmap/issues/482

Before discovering this, I also created this topic on the community forum: https://github.community/t/projects-beta-api-how-to-get-custom-field-data/259539
